### DISCUSSION TITLE
Preparing the v7.0.5 release

### DISCUSSION THREAD

**Discussion by vmiklos** [vmiklos/plees-tracker#84] [General]
Hi,

I plan to release v7.0.5 towards the end of the next week. It currently contains some English-only strings.

@sguinetti @fabianski7 may I ask you to update the translations you already created in the past?

Thanks,

Miklos

**Comment by fabianski7** [comment]
For sure.
Would you have any version compiled with the recent changes? that would be useful to test the quality of the translation

**Comment by vmiklos** [comment]
https://share.vmiklos.hu/plees-tracker/app-debug.apk is a snapshot of current master. Note that you'll have to uninstall the fdroid apk first, as the signing key is different. (Be sure to export your sleeps first to avoid data loss.)
